src/ocamlorg_frontend/components/header.eml

```diff
@@ -52,8 +52,14 @@ let render
                     d="M16.32 14.9l5.39 5.4a1 1 0 0 1-1.42 1.4l-5.38-5.38a8 8 0 1 1 1.41-1.41zM10 16a6 6 0 1 0 0-12 6 6 0 0 0 0 12z" />
               </svg>
             </button>
-            <div class="header__search__dropdown absolute w-full mt-16 px-4 py-1 mt-20 bg-white text-primary-600 font-semibold border b-1 border-primary-600">
-              <a class="hover:underline" href="<%s Url.api %>">Standard Library API</a>
+            <div class="header__search__dropdown absolute rounded-b-md w-full top-0 mt-10 p-2 bg-white font-semibold border border-primary-600">
+              <span class="pl-2">Or go to:</span>
+              <a class="flex py-2 px-4 gap-4 hover:bg-primary-100 font-regular hover:font-semibold text-primary-600" href="<%s Url.api %>">
+                Standard Library API
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+                  <path stroke-linecap="round" stroke-linejoin="round" d="M13.5 6H5.25A2.25 2.25 0 003 8.25v10.5A2.25 2.25 0 005.25 21h10.5A2.25 2.25 0 0018 18.75V10.5m-10.5 6L21 3m0 0h-5.25M21 3v5.25" />
```

Comment:
Can't you already put it `Icons.ml` I prefer resolving a conflict than searching for SVG tags again :-)

Reply:
Reason being that we might already have this icon in #865

Reply:
Or we merge #865 first. The benefits of that cleanup are worth it.